### PR TITLE
Create chess-bounty.md

### PR DESCRIPTION
**Pull request by thelifeandtimes**
@ashelkovnykov take a look at this and see what you think. Could be good to add a bit about expected timing, how you will work with grantees (frequency of meeting, code reviews, etc), and maybe something about if it is acceptable for a single contributor to take on?

### CODE REVIEW

File: content/grants/chess-bounty.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+title = "Chess Improvements"
+date = "2022-08-22"
+
+[taxonomies]
+grant_type = [ "Bounty" ]
+grant_category = [ "App Dev" ]
+
+[extra]
+image = ""
+description = "Improvements for the Chess app to create a Chess experience comparable to that of any modern Chess website"
```
Comment (ashelkovnykov) [inline content/grants/chess-bounty.md]: "Improvements to the Chess app, creating an Urbit Chess experience comparable to that of any modern Chess website"

File: content/grants/chess-bounty.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+reward = "3 stars"
+assignee = [""]
+grant_id = "B0175"
+champion = "~finmep-lanteb"
+completed = false
+work_request_link = "https://airtable.com/shr4qt9t9kz7RaOIa?prefill_Grant+ID=B0175&prefill_Grant+Name=Chess%3A%20Improved%2C%20Timed%2C%20%26%20Rated"
++++
+
+# Rationale
+~finmep-lanteb is sponsoring the improvement of the Chess app to create a Chess experience comparable to that of any modern Chess website. The original development of Chess on urbit functions in the context of game logic and ability to play with friends, but lacks certain features, such as notifications of your opponent completing their turn, which are necessary to present a complete app experience. These quality of life improvements offer a great opportunity for a team of 3 novice Urbit developers (preferably three Hoon School Live and App School Live graduates) to work together to complete this bounty, each earning a star in the process.
```
Comment (ashelkovnykov) [inline content/grants/chess-bounty.md]: "~finmep-lanteb is sponsoring the improvement of the Chess app to create an Urbit Chess experience comparable to that of any modern Chess website. The current Chess app meets the basic requirements to play a game with a friend, but lacks various quality of life features: move notifications, undo requests, `%pals` integration, etc. In addition, only unrated, correspondence games are currently possible. These missing features & improvements offer a great opportunity for a team of 3 novice Urbit developers (preferably three Hoon School Live and App School Live graduates) to work together to complete this bounty, each earning a star in the process."

File: content/grants/chess-bounty.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+The goals of the bounty are to resolve the many small quality of life issues currently present in the Chess app, as well as enable timed games and rated games.
```
Comment (ashelkovnykov) [inline content/grants/chess-bounty.md]: "The goals of this bounty are to resolve the quality of life issues present in the Chess app, as well as enable timed and rated games."

File: content/grants/chess-bounty.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+# Deliverables
+## Issue Handling
```
Comment (ashelkovnykov) [inline content/grants/chess-bounty.md]: "Enhanced Chess Experience"

File: content/grants/chess-bounty.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+Resolve all of the currently open issues on the [Chess app GitHub page](https://github.com/ashelkovnykov/urbit-chess/issues). Prior to work starting, the grantee and champion will meet and mutually agree to a specific list of numbered issues will be included in the scope of this deliverable.
```
Comment (ashelkovnykov) [inline content/grants/chess-bounty.md]: "Resolve the open issues on the [Chess app GitHub page](https://github.com/ashelkovnykov/urbit-chess/issues). Prior to starting work, the grantee and champion will meet and mutually agree to the specific list of numbered issues included in the scope of this deliverable."

**Review by ashelkovnykov — COMMENTED** [review]
I left some phrasing nits. There's a lot less there than it seems, but I can't comment on specific sentences, so I just quoted entire paragraphs at a time and made changes inline.

Code Reviews:
The grantees would be encouraged to perform reviews of each other's code. As champion, I will perform code reviews, test all of the submitted changes, and grant final approval by merging the PRs. Since the Foundation is sponsoring the ELO and timing servers, I think it's fair if it appoints a secondary reviewer to represent their interests for those two milestones. Otherwise, I'm perfectly satisfied to take on the above responsibility for those two milestones as well.

Champion Time Commitment:
I'm able and willing to spare an hour per day for the purposes of meetings, design aid, or to otherwise support the bounty. Travel and family matters crop up, so this can instead be measured as "7 hours per week".

Expected Completion Time:
For a team of 3, I picture this taking no more than 2 months (I actually think it would take much less time, but issues come up and overruns happen).

Number of Grantees:
Never say never, but I would **strongly** prefer for 3 novice developers to work on this. This bounty is just begging for people breaking into Urbit/coding to take it on.

**Comment by thelifeandtimes** [comment]
@gordoncc this should be ready for your review and merging.

**Review by gordoncc — APPROVED** [review]

**Pull request merged**